### DISCUSSION TITLE
How strong Kryptor's generated passphrase?

### DISCUSSION THREAD

**Discussion by Bull717** [samuel-lucas6/Kryptor#74] [General]
hi, What's the entropy of Kryptor's generated 8 word passphrase? İs it secure enough for future proofing?
thanks!

**Comment by samuel-lucas6** [comment]
Hi, sorry for the delayed reply. According to the [EFF](https://www.eff.org/deeplinks/2016/07/new-wordlists-random-passphrases), each word in their word list has about ~12.9 bits of entropy, meaning an 8 word passphrase has ~103 bits of entropy. Combined with the slowdown from Argon2, this should be good enough for future proofing. However, 10 or 12 words would obviously be even better and so would using a keyfile alongside the password to reach a 128-bit+ security level.
